parser/src/main/java/parser/BookParser.java

```diff
@@ -0,0 +1,29 @@
+package parser;
+
+import org.jsoup.nodes.Document;
+import parser.DTO.Book;
+
+/**
+ * Created by Grzesiek on 2016-08-28.
+ */
+public abstract class BookParser implements IBookParser {
```

Comment:
Is there a need to implement IBookParser here? Are you going to have more than one class that implements this interface? BookParser is an abstract class, maybe it is enough?